packages/state-manager/src/sagas/messages/messages.slice.ts

```diff
@@ -38,6 +39,10 @@ export const messagesSlice = createSlice({
   name: StoreKeys.Messages,
   reducers: {
     sendMessage: (state, _action: PayloadAction<WriteMessagePayload>) => state,
+    deleteMessages: (state, action: PayloadAction<DeleteMessagesPayload>) => {
```

Comment:
I think this name may be confusing. It sounds like it deletes certain amount of messages, How about marking that we delete the whole channel entry for messages?